packages/jaeger-ui/tsconfig.lint.json

```diff
@@ -28,11 +28,29 @@
   "files": [
```

Comment:
this feels kinda insane that we need to list the files like this. Isn't there a way to use a glob like `**/*.js` ?

Reply:
That works as well. I just wanted to be explicit here (ideally I'd imagine this list will eventually go away anyways as things are converted to TS).

Reply:
my concern is more about missing files here and, as a result, accumulating more lint issues over time.

Reply:
AIUI `tsc` won't actually lint these files, since they are JS and we don't have `checkJs` specified in `compilerOptions`—it will however complain if they are absent and a TS file tries to import them.